tests/Model/FIT/GlobalProfileTest.php

```
@@ -3,117 +3,20 @@
-        if(array_key_exists('components', $field)){
-          foreach($field['components'] as $component){
-            $message = "Checking message type: "
-              . $message_type['name']
-              . " | field: " . $field['name']
-              . " | component " . $component['name']
-            ;
-            $this->assertNotContains($component['name'], $component_names, $message);
-            $component_names[] = $component['name'];
-          }
+    foreach(GlobalProfileAccess::getAllFieldDefinitions() as $field_definition){
+      $field_def_num = $field_definition->getNumber();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$field_def_num'. (Unused Code Rules)

(UnusedLocalVariable)